src/sentry/api/endpoints/group_details.py

```diff
@@ -206,24 +199,44 @@ def get(self, request, group):
         if last_release:
             last_release = self._get_release_info(request, group, last_release)
 
-        try:
-            environment_id = self._get_environment_id_from_request(
-                request, group.project.organization_id)
-        except Environment.DoesNotExist:
-            get_range = lambda model, keys, start, end, **kwargs: \
-                {k: tsdb.make_series(0, start, end) for k in keys}
-            tags = []
-            user_reports = UserReport.objects.none()
+        # TODO(jess): This can be removed when tagstore v2 is deprecated
+        use_snuba = (
+            settings.SENTRY_TAGSTORE == 'sentry.tagstore.snuba.SnubaTagStorage' or
```

Comment:
this also seems gross, but i just want a way for this to follow whatever we are generally doing in production for snuba use

Reply:
yeah given that tagstore is getting deprecated ASAP, but we might not be ready to switch over these group details queries to use snuba in prod right this moment, I'd probably put it behind the `snuba.events-queries.enabled` flag, or some other flag (`snuba.group-queries.enabled` ?) that controls it independently of tagstore.